case-studies/wink.html

```diff
@@ -0,0 +1,125 @@
+---
+title: WINK Case Study
+layout: none
+class: gridPage
+cid: caseStudies
+---
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+      <link rel="stylesheet" type="text/css" href="/css/style_wink.css">
+    <title>WINK Case Study</title>
+  </head>
+<body>
+
+
+<div class="banner1">
+  <h1>CASE STUDY: <img src="/images/wink_logo.png" class="logo" width="13%" style="margin-bottom:-4px"><br>
+    <div class="subhead">Cloud-Native Infrastructure Keeps Your Smart Home Connected</div>
+  </h1>
+</div>
+
+
+<div class="details">
+    Company &nbsp;<b>Wink</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Location &nbsp;<b>New York, N.Y.</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Industry &nbsp;<b>Internet of Things Platform</b>
+</div>
+
+<hr>
+
+<section class="section1">
+
+<div class="cols">
+  <div class="col1">
+
+      <h2>Challenge</h2>
+        Building a low-latency, highly reliable infrastructure to serve communications between millions of connected smart-home devices and the company’s consumer hubs and mobile app, with an emphasis on horizontal scalability, the ability to encrypt everything quickly and connections that could be easily brought back up if anything went wrong.
+        <br>
+
+      <h2>Solution</h2>
+        Across-the-board use of a Kubernetes-Docker-CoreOS Container Linux stack.<br><br>
+  </div>
+
+  <div class="col2">
+    <h2>Impact</h2>
+      "Two of the biggest American retailers [Home Depot and Walmart] are carrying and promoting the brand and the hardware,” Wink Head of Engineering Kit Klein says proudly – though he adds that "it really comes with a lot of pressure. It’s not a retail situation where you have a lot of tech enthusiasts. These are everyday people who want something that works and have no tolerance for technical excuses.” And that’s further testament to how much faith Klein has in the infrastructure that the Wink team has built. With 80 percent of Wink’s workload running on a unified stack of Kubernetes-Docker-CoreOS, the company has put itself in a position to continually innovate and improve its products and services. Committing to this technology, says Klein, "makes building on top of the infrastructure relatively&nbsp;easy.”
+  </div>
```

Comment:
I don't understand what brand and hardware this refers to. When I read the same text later in the article, then I do understand. But here, out of context, I don't.